apps/blog/src/posts/announcing-cli/announcing-cli.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Use the PWABuilder CLI to Create a New Progressive Web App
+excerpt: Announcing PWABuilder's new CLI. Create, develop, and test from an easy-to-use PWA template.
+description: Announcing PWABuilder's new CLI. Create, develop, and test from an easy-to-use PWA template.
+date: 2023-04-19
+updatedDate: 2023-04-19
+trending: true
+featured: true
+image: 
+isPost: true
+backUrl: '/'
+author:
+  name: Zach Teutsch
+  twitter: https://twitter.com/devteutsch
+  title: PWABuilder Engineer
+tags:
+  - CLI
+  - PWA Starter
+  - PWA
+  - PWA template
+---
+
+PWABuilder is launching a brand new CLI that can create a new templated progressive web app for you and have you developing in just seconds. 
```

Comment:
Capitallize progressive web app, plus maybe a link to our intro to pwa docs?

Reply:
```suggestion
PWABuilder is launching a brand new CLI that can create a new templated [Progressive Web App](https://docs.pwabuilder.com/#/home/pwa-intro) for you and have you developing in just seconds. 
```